css/jspage.css

```
@@ -0,0 +1,200 @@
+@import url('https://fonts.googleapis.com/css2?family=Libre+Caslon+Text:wght@700&family=M+PLUS+1:wght@700&family=Oxygen:wght@300;400&family=Ubuntu&display=swap');
+* {
+  margin: 0;
+  padding: 0;
+ 
+}
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-family is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)